src/components/PeopleAndRoles/PeopleAndRoles.vue

```diff
@@ -152,18 +151,34 @@ export default class PeopleAndRoles extends Mixins(EntityFilterMixin) {
     }
   }
 
+  // Local properties
   private showOrgPersonForm: boolean = false
   private activeIndex: number = -1
   private addEditInProgress: boolean = false
   private currentOrgPerson: OrgPersonIF | null = null
   private nextId: number = -1
 
+  // Enums for the template
   readonly EntityTypes = EntityTypes
   readonly Roles = Roles
   readonly IncorporatorTypes = IncorporatorTypes
 
+  /** Whether to show errors. */
+  private get showErrors (): boolean {
+    return Boolean(this.$route.query.showErrors)
+  }
```

Comment:
how exactly does this work?

Reply:
If the route has a query parameter named "showErrors" (and it's true) then this returns `true`.